domain/schema/model/sql/0022-resource.sql

```diff
@@ -133,3 +134,13 @@ CREATE TABLE unit_resource (
     REFERENCES unit (uuid),
     PRIMARY KEY (resource_uuid, unit_uuid)
 );
+
+CREATE TABLE docker_metadata {
```

Comment:
**suggestion:** I think the name of the table is prone to decay. Since it is a 0..1 relationship with resource table, it should be prefixed by `resource_`:

* Better lookup in IDE
* Avoid potential futur clashes with other kind of "docker metadata" we may need in the futur (nobody knows)

**nitpick:** `docker`  isn't the go-to name for oci image anymore. Even if it make the name of the table very long, I would softly vouch for a more general name `resource_oci_image_metadata` or even, to express better the intent: `resource_oci_image_metadata_store` (because it is actually a hack to avoid putting plain text password in the object store... Another approach would be to rely on some secret handling with injection into resources, or having a trusted storage)